### ISSUE TITLE
Handle 250+ items in an invoice

### ISSUE THREAD

**Issue by nickoferrall** [ParabolInc/parabol#6827]
[Stripe limits](https://stripe.com/docs/api/invoiceitems/create) the number of items in an invoice to 250. A seat change counts as two so we can have 125 add/removes in an invoice. 

See [this Slack thread](https://parabol.slack.com/archives/C836NA350/p1655858819881809) for more info.

Some ideas on how we could handle this:

1. Limit the number of users in a Pro org to 125. If more users try to sign-up, ask them to upgrade to Enterprise. 

    I won't share the exact numbers in GitHub, but a sizeable chunk of Pro orgs have more than 125 users. This restriction might deter free users from upgrading to Pro. I don't think minor technical restrictions should determine our pricing. 

2. Send multiple invoices. If there are 250 items in an invoice, invoice "June 2022" becomes "June 2022 Part 1" and "June 2022 Part 2". I think this is the most graceful approach, but the refactoring would be more involved as a Stripe invoice would be changed from one to many.

3. Contact Stripe and ask for an increase in invoice limits. No harm in trying, but I imagine we're not a big enough account to get special treatment.
4. In [generateInvoice](https://github.com/ParabolInc/parabol/blob/7c60878ee1740fd9b60a1d62c10b584d04802d11/packages/server/billing/helpers/generateInvoice.ts), if there are more than 250 items, the 250th item could read: "And X other users (contact support for their emails)", with the sum total of all users that couldn't fit into the invoice. "And X other users" shouldn't include both added & removed users. If there are more than 250 added users and several removed users, the "And X other users" item should be in both sections.

Option 4 is my favourite approach as it's the simplest. It's not ideal that we specify some but not all users in an invoice, but at current usage numbers, it's unlikely we'll run into this problem very often. To my knowledge, this has only occurred once after we made some accidental database changes to Parabol's org. So, option 2 feels over-engineered. 

If it did happen, it'd probably be a good thing for them to contact support as we'd like to know why there's an exodus/influx of users.

**Comment by mattkrick** [comment]
Worth noting that we don't have to limit the number of members, just the number of changes per month. If they grow by 50 members/month, that's totally reasonable!

Could you provide some more detail how option 4 would work? Say a user joins the organization making for 251 invoice items. Would you delete the 2 new invoice items & add their numbers onto an existing item? Would there still be a way to track who on the team joined/left?

I like option 2 as well! I only scratched the surface, but it looks like creating a 1-off invoice shouldn't be too complicated: https://stripe.com/docs/billing/invoices/subscription#pending-items-frequently

Question for @avivapinchas i know you mentioned doing away with proration, but do we expect to move away from a per-seat billing model for pro? just want to make sure before we dig into this work.

**Comment by avivapinchas** [comment]
> do we expect to move away from a per-seat billing model for pro? 

I don't expect to move away from per-user pricing. 

> Option 4 is my favourite approach as it's the simplest

FWIW - I'm in favor of anything that simplifies the invoice. I _would_ like to look at the design of the invoice itself because I personally think it's hard to parse, too much detail - we have a couple of issues on this: #6575, #6571

This doesn't have to be a blocker for this issue, just sharing for context, in case it sways which direction y'all want to go in.

**Comment by mattkrick** [comment]
Good to know! I _think_ we still want to maintain the ability to connect a charge to the activity of a particular user, even if we don't show it. I'd hate for someone with strict billing requirements to ask us how we arrived at the total & not have a firm answer.

**Comment by avivapinchas** [comment]
Yep, totally agreed!

**Comment by mattkrick** [comment]
closing in favor of #6810
